include/seqan3/alignment/pairwise/policy/affine_gap_init_policy.hpp

```diff
@@ -64,109 +64,103 @@ class affine_gap_init_policy
      * \tparam        cell_t       The underlying cell type.
      * \tparam        cache_t      The type of the cache.
      * \param[in,out] current_cell The current cell in the dynamic programming matrix.
-     * \param[in,out] cache        The cache storing hot helper variables.
+     * \param[in]     cache        The cache storing hot helper variables.
      */
     template <typename cell_t, typename cache_t>
     constexpr auto init_origin_cell(cell_t && current_cell, cache_t & cache) const noexcept
```

Comment:
if `cache` is an `in` paramter why is it not const?

Reply:
I'll guess that can be const then

Reply:
Nop, that is in fact an in,out parameter